### PR TITLE
Changes in signature of the function of FastBridgeTransaction

### PR DESCRIPTION
**Pull request by pamgonzalez**

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/peg/Bridge.java [hunk 1]
```diff
@@ -1094,20 +1095,20 @@ public boolean hasBtcBlockCoinbaseTransactionInformation(Object[] args) {
...
+        logger.trace("registerFastBridgeBtcTransaction");
 
         try {
-            return bridgeSupport.registerBtcTransfer(
+            byte[] btcTxSerialized = (byte[]) args[0];
+            int height = ((BigInteger) args[1]).intValue();
+            byte[] pmtSerialized = (byte[]) args[2];
+            Sha256Hash derivationArgumentsHash = Sha256Hash.wrap((byte[]) args[3]);
+            Address userRefundAddress = new Address(bridgeConstants.getBtcParams(), (byte[]) args[4]);
+            RskAddress lbcAddress = new RskAddress(((DataWord)args[5]).getLast20Bytes());
```
Comment (ajlopezrsk) [inline rskj-core/src/main/java/co/rsk/peg/Bridge.java:1107]: Coukld we check if the argument is really an address? Now, the code discard the "left-most" bytes, that could contain anything.

File: rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java [hunk 2]
```diff
@@ -577,14 +577,14 @@
             activations -> activations.isActive(RSKIP143),
             false
     ),
-    REGISTER_BTC_TRANSFER(
+    REGISTER_FAST_BRIDGE_BTC_TRANSACTION(
```
Comment (ajlopezrsk) [inline rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java:580]: Why this change is NOT A HARD FORK? AFAIK, this change doesn't rerun previous sent transactions

File: rskj-core/src/main/java/co/rsk/peg/Bridge.java (hunk 1 above)
Comment (pamgonzalez) [reply rskj-core/src/main/java/co/rsk/peg/Bridge.java:1107]: I don't expect more than 20 bytes, since the "address" type of solidity is internally a 20-byte array. When DataWord is cast for handling, it is converted to a 32-byte array by setting 0s to it. It may be that I am not seeing something that you are.

File: rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java (hunk 2 above)
Comment (pamgonzalez) [reply rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java:580]: It is a HF. (RSKIP 176)

File: rskj-core/src/main/java/co/rsk/peg/Bridge.java (hunk 1 above)
Comment (pamgonzalez) [reply rskj-core/src/main/java/co/rsk/peg/Bridge.java:1107]: I will modify it! thank you!

File: rskj-core/src/main/java/co/rsk/peg/Bridge.java [hunk 3]
```diff
@@ -1116,10 +1117,11 @@ public int registerBtcTransfer(Object[] args) {
                 userRefundAddress,
                 lbcAddress,
                 lpBtcAddress,
-                executionStatus);
-        } catch (BlockStoreException | RegisterBtcTransferException | IOException | RegisterBtcTransactionException e) {
-            logger.warn("Exception in registerBtcTransfer", e);
-            throw new RuntimeException("Exception in registerBtcTransfer", e);
+                valueToTransfer
+            );
+        } catch (BlockStoreException | RegisterFastBridgeBtcTransactionException | IOException | RegisterBtcTransactionException e) {
```
Comment (guidohernan93) [inline rskj-core/src/main/java/co/rsk/peg/Bridge.java:1122]: RegisterBtcTransactionException should not be needed I think

**Review by guidohernan93 — APPROVED** [review]

**Review by josedahlquist — APPROVED** [review]
